english/bob-barker/1970s/1970s.md

```diff
@@ -2,20 +2,20 @@ You open your eyes and see yourself reflected back in a mirror.
 
 You have on a bright blue shirt whose collar is about twice the size it should be.  
 
-You look down and see that your jeans are bell bottoms fastended with an oversized buckle.
+You look down and see that your jeans are bell bottoms fastened with an oversized buckle.
 
 You mumble, "What in the world?"
 
 It's at that moment you realize a man is standing directly next to you.  
 
-He reminds you of a cariacture of a cariacture with crazy glasses.
+He reminds you of a caricature of a caricature with crazy glasses.
 
 "Andy Warhol?", you ask tentatively.
 
-"Yeah baby!", he replies.  "Time to a go-go!"
+"I think it would be terrific if everybody was alike.", he replies.  "Fantasy love is much better than reality love. Everyone needs a fantasy."
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.